### PR TITLE
applications/callflow: Added  set_alert_info

### PR DESCRIPTION
**Pull request by sergey-safarov**

### CODE REVIEW

File: applications/callflow/src/module/cf_set_alert_info.erl [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+%%------------------------------------------------------------------------------
+%% @doc Entry point for this module
+%% @end
+%%------------------------------------------------------------------------------
+-spec handle(kz_json:object(), kapps_call:call()) -> 'ok'.
+handle(Data, Call) ->
+    Value = kz_json:get_ne_binary_value(<<"alert_info">>, Data),
+    set_variable(Value, Call),
+    Call1 = kapps_call:insert_custom_channel_var(<<"Alert_info">>, Value, Call),
```
Comment (jamesaimonetti) [inline applications/callflow/src/module/cf_set_alert_info.erl]: Why not just use `kapps_call:set_custom_channel_var/3` which should set the var on the channel and update the call record's CCVs?

Comment (sergey-safarov) [reply applications/callflow/src/module/cf_set_alert_info.erl]: Hello James
Your suggestions is applied.

File: applications/callflow/src/module/cf_set_alert_info.erl [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+-export([handle/2]).
+
+%%------------------------------------------------------------------------------
+%% @doc Entry point for this module
+%% @end
+%%------------------------------------------------------------------------------
+-spec handle(kz_json:object(), kapps_call:call()) -> 'ok'.
+handle(Data, Call) ->
+    Value = kz_json:get_ne_binary_value(<<"alert_info">>, Data),
```
Comment (sergey-safarov) [inline applications/callflow/src/module/cf_set_alert_info.erl]: Hello James (@jamesaimonetti)
Is `undefined` value will be filtered before sending message to RMQ?
Or need to add `case` here?

**Comment by bazemoreb** [comment]
Hi all - just wanted to see what the latest was here.  Anything needed to accept the PR?

**Comment by bazemoreb** [comment]
Pinging @jamesaimonetti here to see if we might be able to get this one approved?  We were hoping to have prior to cluster upgrade.  thanks!

**Comment by jamesaimonetti** [comment]
@sergey-safarov @bazemoreb sorry, I haven't been getting Github notifications! Merged.

**Comment by bazemoreb** [comment]
No problem, thanks!  You think it will get in to a release by Friday-ish?

Thanks,

Brandon Bazemore
Audian

844.611.6110 – TF                  Kirkland HQ
425.629.1234 – Direct            11232 120th Ave NE #206
844.737.7375 – Fax                Kirkland, WA 98033

[cid:image001.png@01D395F2.012E1D60]

**Comment by jamesaimonetti** [comment]
Not sure, but you can certainly file a support ticket to find out from someone with more knowledge there.

**Comment by sergey-safarov** [comment]
Hello Brandon (@bazemoreb)
Created PRs https://github.com/2600hz/kazoo/pull/4912 and https://github.com/2600hz/kazoo/pull/4913 for Audian needs

**Pull request closed without merging**